podvm/Dockerfile.podvm

```diff
@@ -8,7 +8,7 @@ ARG BUILDER_IMG="quay.io/confidential-containers/podvm-builder-ubuntu"
 
 FROM ${BUILDER_IMG} AS podvm_builder
 
-ARG CLOUD_PROVIDER=libvirt
+ARG CLOUD_PROVIDER
```

Comment:
Any particular reason for removing the default?

Reply:
TBH it's not tightly coupled with this PR, removing the default makes the container runtime to print a warning if the variable is not set, just to avoid mistakes, nothing else

Reply:
I'd prefer to leave it set, but not a strong perference

Reply:
Forcing the user to set CLOUD_PROVIDER will help with preventing bugs because someone forgot to set CLOUD_PROVIDER with the right target for the image. (However, as of today, I don't think there are any such dependenices for the image. )